CMakeLists.txt

```diff
@@ -143,6 +143,19 @@ if(NOT CLANG_RESOURCE_DIR)
       break()
     endif()
     set(tried "${tried}\n  ${d}/clang/${v}")
+    
+    # On Gentoo clang resources are installed side-by-side with llvm: 
+    #   /usr/lib64/llvm/...
+    #   /usr/lib64/clang/... 
```

Comment:
What is the actual value of `LLVM_LIBRARY_DIRS`?  What is the actual path found as `CLANG_RESOURCE_DIR` with this change?

Reply:
LLVM_LIBRARY_DIRS = /usr/lib64/llvm/5/lib64 (What's returned by llvm-config --libdir)
CLANG_RESOURCE_DIR=/usr/lib/clang/5.0.1/